src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
+    static String horizontalLine = "--------------------------------------------------";
+    static int thingsCounted = 0;
+    static Task[] tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(horizontalLine + "\nHello! I'm Duke\n" + "What can I do for you?\n" + horizontalLine);
+        storeTextAndList();
+    }
+
+    public static void storeTextAndList() {
+        String text;
```

Comment:
Variables should be initialized where they are declared to ensure that it is valid at any time, consider moving it to line 16 so it's "String text = in.nextLine();".